apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -504,26 +500,20 @@ void CameraPipe::generate() {
         Expr out_width = processed.width();
         Expr out_height = processed.height();
 
-        // In HVX 128, we need 2 threads to saturate HVX with work,
-        //and in HVX 64 we need 4 threads, and on other devices,
-        // we might need many threads.
+        // In HVX 128, we need 4 threads to saturate HVX with work,
+        // on other devices, we might need many threads.
         Expr strip_size;
-        if (get_target().has_feature(Target::HVX_128)) {
-            strip_size = processed.dim(1).extent() / 2;
-        } else if (get_target().has_feature(Target::HVX_64)) {
+        if (get_target().has_feature(Target::HVX)) {
             strip_size = processed.dim(1).extent() / 4;
```

Comment:
Keep the HVX_128 branch, not the HVX_64 branch.

Reply:
I jumped the gun a little bit here. Newer cores have 4 contexts, but presently the lowest we support still has 2 contexts. I'll switch it back to `` / 2 ;``

Reply:
Oh, I see. Actually, I think having more strips than cores is fine, so maybe keeping this 4 is best.